Common/Utility/DocDbRestUtility.cs

```diff
@@ -267,6 +270,26 @@ public DocDbRestUtility(IConfigurationProvider configProvider)
             return JObject.Parse(response);
         }
 
+        public async Task<DeviceND> SaveNewDocumentAsyncND(DeviceND document)
```

Comment:
lol we must've forgot. Let's keep using this cause people added stuff in this class before me.